website/docs/d/cloudinit_config.html.markdown

```diff
@@ -52,6 +52,15 @@ resource "aws_instance" "web" {
   instance_type = "t2.micro"
   user_data     = "${data.template_cloudinit_config.config.rendered}"
 }
+
+# Start a GCP instance with the cloudinit config as user data.
```

Comment:
Add a note that one has to disable gzipping and base64 encoding with:
```
  gzip          = false
  base64_encode = false
```
as otherwise, the supplied user-data will be just ignored.